index.js

```diff
@@ -69,10 +69,26 @@ function join(array, string) {
   //your code here
 }
 
+// Now let's make something new entirely! We're going to make filterMap.
+//
+// This is a function which takes an array, a function which returns a boolean,
```

Comment:
This sentence took me a few tries to understand. How about laying it out in a list?
eg
// The function filterMap takes 3 arguments:
- an array
- a function that returns a boolean,
- a function that maps the array element.